tracee/tracee.go

```diff
@@ -341,6 +341,51 @@ func (t *Tracee) processEvent(ctx *context, args []interface{}) error {
 		}
 	}
 
+	//capture executed files
+	if t.config.CaptureFiles && (eventName == "execve") {
```

Comment:
You can use security_bprm_check instead and catch execveat as well

Reply:
will do